pages/portfolio.html

```diff
@@ -4,9 +4,34 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Cabebe's Portfolio</title>
+  <link href="/styles/styles.css" rel="stylesheet">
 </head>
-<body>
-  
+<body id="container">
+  <nav id="nav-block">
+    <ol id="nav-bar" class="flex-box">
+      <li><a id="first-nav-link" class="nav-links" href="index.html">Home</a></li>
+      <li><a class="nav-links" href="about.html">About</a></li>
+      <li><a class="nav-links" href="portfolio.html">Portfolio</a></li>
```

Comment:
If you moved the index.html file to the root of the projects the paths of the files would like the following:
```suggestion
<body id="container">
  <nav id="nav-block">
    <ol id="nav-bar" class="flex-box">
      <li><a id="first-nav-link" class="nav-links" href="./pages/index.html">Home</a></li>
      <li><a class="nav-links" href="./pages/about.html">About</a></li>
      <li><a class="nav-links" href="./pages/portfolio.html">Portfolio</a></li>
```